Makefile

```diff
@@ -27,9 +27,9 @@ generate-copy:
 
 generate-client:
 	rm -rf maistra.io
-	$(CLIENT_GEN) --clientset-name versioned --input-base "" --input $(kube_api_packages) --output-package $(kube_clientset_package) -h $(header_file) --output-base ./
-	$(LISTER_GEN) --input-dirs $(kube_api_packages) --output-package $(kube_listers_package) -h $(header_file) --output-base ./
-	$(INFORMER_GEN) --input-dirs $(kube_api_packages) --versioned-clientset-package $(kube_clientset_package)/versioned --listers-package $(kube_listers_package) --output-package $(kube_informers_package) -h $(header_file) --output-base ./
+	$(CLIENT_GEN) --clientset-name versioned --input-base "" --input $(kube_api_packages) --output-package $(kube_clientset_package) -h $(header_file) --output-base ./ --plural-exceptions ServiceExports:ServiceExports
+	$(LISTER_GEN) --input-dirs $(kube_api_packages) --output-package $(kube_listers_package) -h $(header_file) --output-base ./ --plural-exceptions ServiceExports:ServiceExports
+	$(INFORMER_GEN) --input-dirs $(kube_api_packages) --versioned-clientset-package $(kube_clientset_package)/versioned --listers-package $(kube_listers_package) --output-package $(kube_informers_package) -h $(header_file) --output-base ./ --plural-exceptions ServiceExports:ServiceExports
 	rm -rf client && mv maistra.io/api/client . && rm -rf maistra.io
```

Comment:
Just out of curiosity, why aren't we using the `generate-groups.sh` script, as opposed to invoking each of the generators individually?

Reply:
No strong reason. I guess explicit is better than explicit? Perhaps it gives us more flexibility? Do you prefer to call the script instead?